boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -182,12 +186,25 @@ your Zephyr application to flash. It will leave you at a GDB prompt.
    :board: mimxrt1050_evk
    :goals: debug
 
+Board Revisions
+***************
+
+MIMXRT1050-EVKB is revison A1, while MIMXRT1050-EVK board is rev. A0. There are
```

Comment:
Better would be:

    The original MIMXRT1050-EVK (rev A0) board was updated with a newer
    MIMXRT1050-EVKB (rev A1) board, with these major hardware differences: